<launch>
	<arg name="camera" default="camera" />
	<arg name="auto_exposure" default="false" />
	<arg name="auto_white_balance" default="false" />

	<include file="$(find openni2_launch)/launch/openni2.launch">
		<arg name="camera" default="$(arg camera)" />
	</include>

	<group ns="carote">
		<node name="target" pkg="carote" type="target" respawn="true" output="screen">
			<param name="input_sensor" value="/$(arg camera)"/>
			<param name="input_frame" value="camera_depth_optical_frame"/>
			<param name="input_cloud" value="/depth_registered/points"/>
			<param name="output_image" value="/image"/>
			<param name="output_position" value="/position"/>
			<param name="output_velocity" value="/velocity"/>
		</node> 

		<!--
		<node name="carote_parameters" pkg="rqt_reconfigure" type="rqt_reconfigure" respawn="false" output="screen"/>
		-->
	</group>
</launch>
